app/assets/src/components/views/amr_heatmap/amr_heatmap_vis.scss

```diff
@@ -19,4 +19,28 @@
       display: block;
     }
   }
+
+  .legend {
+    padding: 10px;
+    background-color: $white;
+    border: 1px solid $light-grey;
+    border-radius: 5px;
+    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.2);
```

Comment:
you should use one of the box-shadows in _elements.scss or ask Jenn if you need to define a new one